### PR TITLE
Added icon images, removed unnecessary images and fixed bugs with overlapping statusbar

### PR DESCRIPTION
**Pull request by Interperle**
Please have a look if I destroyed anything for android

Received new images by @jannikbmc 

Signed-off-by: Interperle <marib.aldoais@gmail.com>

### CODE REVIEW

**Review by jannikbmc — APPROVED** [review]

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj [hunk 1]
```diff
@@ -13,9 +13,9 @@
   <ItemGroup>
     <PackageReference Include="Plugin.BLE" Version="2.1.3" />
     <PackageReference Include="sqlite-net-pcl" Version="1.8.116" />
-    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2012" />  
-    <PackageReference Include="Xamarin.Essentials" Version="1.6.1" />  
-    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2012" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.2478" />  
+    <PackageReference Include="Xamarin.Essentials" Version="1.7.3" />  
+    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2478" />
```
Comment (Chippelius) [inline Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj:18]: If you want to change the Xamarin-Versions, you not only need to adapt the iOS-Setup, but also the android-Setup accordingly.

**Comment by Chippelius** [comment]
And I just triggered a run of our CI pipelin on this branch: the unit tests fail: 
https://github.com/amosproj/amos2022ss05-find-my-hearing-aid/actions/runs/2673415471

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/AboutPage.xaml [hunk 2]
```diff
@@ -30,7 +32,7 @@
                                  Padding="0,20,0,45" 
                                  HeightRequest="220" 
                                  Source="signal_strength_about.png"
-                                 Aspect="AspectFit" CornerRadius="50" 
+                                 Aspect="Fill" CornerRadius="50"
```
[the remark this answers is not in the document]
Comment (Interperle) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/AboutPage.xaml:35]: done

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/AboutPage.xaml [hunk 3]
```diff
@@ -52,7 +54,7 @@
                                  Padding="0,20,0,45" 
                                  HeightRequest="220"                                 
                                  Source="geolocation_about.png"                         
-                                 Aspect="AspectFit" CornerRadius="50" 
+                                 Aspect="Fill" CornerRadius="50"
```
[the remark this answers is not in the document]
Comment (Interperle) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/Views/AboutPage.xaml:57]: done

File: Implementation/FindMyBLEDevice.Tests/FindMyBLEDevice.Tests.csproj [hunk 4]
```diff
@@ -15,9 +15,9 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2012" />
+    <PackageReference Include="Xamarin.Forms.Maps" Version="5.0.0.2478" />
     <PackageReference Include="Xamarin.Forms.Mocks" Version="4.7.0.1" />
-	<PackageReference Include="Moq" Version="4.18.0" />
+	<PackageReference Include="Moq" Version="4.18.1" />
```
[the remark this answers is not in the document]
Comment (Interperle) [reply Implementation/FindMyBLEDevice.Tests/FindMyBLEDevice.Tests.csproj:20]: done

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj (hunk 1 above)
Comment (Interperle) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/FindMyBLEDevice.csproj:18]: done

**Review by Meisterlu — APPROVED** [review]
Approved but Icons in the home page need a higher resolution, this will be fixed by @Interperle

**Comment by Interperle** [comment]
> And I just triggered a run of our CI pipelin on this branch: the unit tests fail:
> https://github.com/amosproj/amos2022ss05-find-my-hearing-aid/actions/runs/2673415471

The Pipeline already fails for ui-rework2: https://github.com/amosproj/amos2022ss05-find-my-hearing-aid/runs/7354274909?check_suite_focus=true#step:5:147

 The errors which occur have nothing todo with my changes.

**Review by Chippelius — APPROVED** [review]

**Pull request merged**